t/api/10-jobgroups.t

```diff
@@ -131,6 +147,18 @@ subtest 'create parent group' => sub() {
 };
 
 subtest 'create job group' => sub() {
+    $t->post_ok(
+        '/api/v1/job_groups',
+        form => {
+            name                                => 'Cool group',
+            size_limit_gb                       => '-300GB',
+            default_keep_important_logs_in_days => '4 days'
+        }
+    )->status_is(400)->json_is(
+        '/error' => 'Invalid request parameters (default_keep_important_logs_in_days, size_limit_gb)',
```

Comment:
is it ensured that the validation error messages always appear on an alphabetically ordered list of parameters?

Reply:
Yes. [Mojolicious::Validator->failed](https://mojolicious.org/perldoc/Mojolicious/Validator/Validation#failed) returns a sorted array (although it's not (yet) documented).